test/root_component_test.dart

```diff
@@ -85,6 +85,9 @@ void main() {
     test(
         'handleClientLoad(), initClient() and loadOffice()'
         'are invoked during init', () async {
+      when(mockCredential.initClient()).thenAnswer((_) => Future.value(true));
```

Comment:
These extra setups are to ensure that [_updatePage()](https://github.com/googleinterns/dv360-excel-plugin/pull/80/commits/a4f2c4b74f028fa998a59a1b9fc6d12e26a49a70#diff-0ddd5d14fdbdacb8aa08fb8299ded715R48) won't throw when the bools are null.